entries/a/americancentury.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "American Century Investments": {
+    "domain": "americancentury.com",
+    "url": "https://www.americancentury.com",
```

Comment:
```suggestion
```

You only need the `url` key if going to `https://domain` (e.g. `https://americancentury.com`) doesn't work

Reply:
Got it :+1: 

Removed!